README.md

```diff
@@ -20,7 +20,7 @@ Or install it yourself as:
 
 ## Usage
 
-You'll need to generate an access token in Digital Ocean's control panel at https://cloud.digitalocean.com/settings/applications
+You'll need to generate an access token in DigitalOcean's control panel at https://cloud.digitalocean.com/settings/applications
 
 With your access token, retrieve a client instance with it.
```

Comment:
Could we fix this sentence, pretty please?

`Using your access token, retrieve a client instance.`

Reply:
Sure, I can do that in a follow-up PR.